lbz/dev/server.py

```diff
@@ -5,22 +5,27 @@
 import json
 import logging
 import urllib.parse
+from abc import ABCMeta, abstractmethod
```

Comment:
wrong import order

Reply:
PyCharm disagrees with You, anyway isort is next on the list.